tests/build-init-files.py

```diff
@@ -0,0 +1,56 @@
+import logging
+import pathlib
+import sys
+
+import click
+
+
+log_levels = {
+    0: logging.ERROR,
+    1: logging.WARNING,
+    2: logging.INFO,
+}
+
+
+@click.command()
+@click.option(
+    "-r", "--root", "root_str", type=click.Path(dir_okay=True, file_okay=False, resolve_path=True), default="."
+)
+@click.option("-v", "--verbose", count=True, help=f"Increase verbosity up to {len(log_levels) - 1} times")
+def command(verbose, root_str):
+    logger = logging.getLogger()
+    log_level = log_levels.get(verbose, min(log_levels.values()))
+    logger.setLevel(log_level)
+    stream_handler = logging.StreamHandler()
+    logger.addHandler(stream_handler)
+
+    tree_roots = ["chia", "tests"]
+    failed = False
+    root = pathlib.Path(root_str).resolve()
+    directories = sorted(
+        path
+        for tree_root in tree_roots
+        for path in root.joinpath(tree_root).rglob("**/")
+        if "__pycache__" not in path.parts
+    )
+
+    for path in directories:
+        init_path = path.joinpath("__init__.py")
+        # This has plenty of race hazards. If it messes up,
+        # it will likely get caught the next time.
+        if init_path.is_file() and not init_path.is_symlink():
+            logger.info(f"Found   : {init_path}")
+            continue
+        elif not init_path.exists():
+            failed = True
+            init_path.touch()
+            logger.warning(f"Created : {init_path}")
+        else:
+            failed = True
+            logger.error(f"Fail    : present but not a regular file: {init_path}", file=sys.stderr)
+
+    if failed:
+        raise click.ClickException("At least one __init__.py created or not a regular file")
+
+
+command()  # noqa: E1120
```

Comment:
Sorry about that, I added a flake8 ignore last time.  This seems to work locally as a pylint ignore...

```suggestion
command()  # pylint: disable=no-value-for-parameter
```